spec/api-web-contents-spec.js

```diff
@@ -799,4 +800,50 @@ describe('webContents module', () => {
       w.loadURL('about:blank')
     })
   })
+
+  describe('takeHeapSnapshot()', () => {
+    const filePath = path.join(remote.app.getPath('userData'), 'test.heapsnapshot')
+
+    afterEach(done => {
+      fs.unlink(filePath, () => done())
```

Comment:
Please call in a `beforeEach` hook.
`afterEach` won't be called for failed tests =/

Reply:
refactored a bit differently